api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
So tags... not indexed on `FileNode` or even `Node.` They are indexed with `elasticsearch` but of course we don't have an API route to that, though it would be interesting if we directed some of the filtering to `elasticsearch` through middleware that could take the filters and then do a search.